From 85017f0a2f9b4d65116c97e6d3c520c3d43e460d Mon Sep 17 00:00:00 2001
From: Mickael Gaillard <mickael.gaillard@tactfactory.com>
Date: Thu, 18 Apr 2019 14:08:45 +0200
Subject: [PATCH 2/2] Add TP for stash

---
 TP7 - Remote.txt | 46 ++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 46 insertions(+)

diff --git a/TP7 - Remote.txt b/TP7 - Remote.txt
index c92c973..e0f26eb 100644
--- a/TP7 - Remote.txt	
+++ b/TP7 - Remote.txt	
@@ -26,4 +26,50 @@ Synchroniser avec votre depot distant
 Modifier le fichier README.md en ajoutant Le nom du formateur
 Conservez un historique avec pour message "Add Mick"
 Synchroniser tous vos depots.
+
+TP8 - Commit Temporaire
+===============================================================================
+
+Modifier le fichier README.md en ajoutant un lorem ipsum...
+Avas de conservez ceci... mettre de coté le lorem ipsul.
+Modifier le fichier README.md en ajoutant le message "Commit intermediaire"
+Conservez un historique avec pour message "use stash"
+Restaurez le lorem ipsum mis de coté !
+Conservez un historique avec pour message "add lorem"
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 
-- 
2.20.1

